components/brave_extension/extension/brave_extension/youtubedown.js

```diff
@@ -0,0 +1,3053 @@
+/* Copyright © 2019-2020 Jamie Zawinski <jwz@jwz.org>
+
+   Permission to use, copy, modify, distribute, and sell this software and its
+   documentation for any purpose is hereby granted without fee, provided that
+   the above copyright notice appear in all copies and that both that
+   copyright notice and this permission notice appear in supporting
```

Comment:
cc @fmarier - do we need to do anything for this?

Reply:
Yes, good catch.

In order to add this license statement onto `brave://credits`, these things need to happen:

1. Create a `components/brave_extension/extension/brave_extension/LICENSE` containing the licensing above (since it appears to be a custom license).
2. Create a `components/brave_extension/extension/brave_extension/README.chromium` file like https://github.com/brave/brave-core/blob/master/components/brave_prochlo/README.chromium pointing to the above `LICENSE` file.
3. Add that file to the list at https://github.com/brave/brave-core/blob/9dbc5f4b93fdddab1eb3aa8980230edc278aba0e/script/brave_license_helper.py#L131.